deployment/src/main/java/io/quarkiverse/fury/deployment/FurySerializerBuildItem.java

```diff
@@ -34,4 +29,17 @@ public Class<?> getClazz() {
     public Class<? extends Serializer> getSerializer() {
         return serializer;
     }
+
+    public static FurySerializerBuildItem buildItem(ClassInfo classInfo) {
+        Class<?> clazz = JandexReflection.loadClass(classInfo);
+        FurySerialization annotation = clazz.getDeclaredAnnotation(FurySerialization.class);
+        if (annotation.targetClass() != FurySerialization.CurrentAnnotatedTypeStub.class) {
+            clazz = annotation.targetClass();
+        }
+        Class<? extends Serializer> serializer = annotation.serializer();
+        if (serializer == Serializer.class) {
+            serializer = null;
+        }
```

Comment:
@chaokunyang  So the `serializer` is null here?

Reply:
It's OK, `Serializer.class` just indicates that no `Serializer` are specified, Serializer should be  picked by Fury itself

Reply:
Then if  `serializer` is null, we don't need to invoke `fury.register(clazz, true);` and replace with
```java
fury.register(clazz, serializer == null);
```

Reply:
removed this check in the new commit